<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BPlot</class>
 <widget class="QDialog" name="BPlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>B-Plot</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,0">
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Setting</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,1">
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBox_Start"/>
      </item>
      <item row="1" column="0" rowspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>x-start</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_EndValue">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" rowspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>x-End</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spinBox_End"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_StartValue">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBox_Integrate">
        <property name="text">
         <string>Integrate</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="5">
    <widget class="QCustomPlot" name="plotArea" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>6</horstretch>
       <verstretch>16</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEdit_Msg">
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="statusTip">
      <string>Save the plot.</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton_Print">
     <property name="text">
      <string>Print as pdf/png</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton_ClearArrows">
     <property name="text">
      <string>Clear Arrows</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
